packages/hash/frontend/src/graphql/queries/user.queries.ts

```diff
@@ -119,9 +120,9 @@ export const logout = gql`
 
 export const meQuery = gql`
   query me {
-    me {
-      ...UserFields
+    me(linkResolveDepth: 3, linkTargetEntityResolveDepth: 3) {
```

Comment:
Good spot, I think you're right that this can be 2 - addressed in https://github.com/hashintel/hash/pull/1280/commits/b90a7fd844ff9a3960e5f6960e4d6bb6cea488af